src/components/information/Information.svelte

```diff
@@ -17,6 +17,7 @@
   //       children, and a component specialised for text. In my opinion (Jon) this should be split up into one general
   //       and one for text (which uses the general one)
   export let iconText: string | undefined = undefined;
+  export let iconTextClass: string | undefined = undefined;
```

Comment:
I didn't use `$$restProps.class` to pass `class` prop down since it might not be obvious which `class` the `class` prop is for seeing as there are multiple elements